src/PhpSpec/Matcher/ThrowMatcher.php

```diff
@@ -121,11 +121,25 @@ public function verifyPositive($callable, array $arguments, $exception = null)
         }
 
         if (!$exceptionThrown instanceof $exception) {
+            $format = 'Expected exception of class %s, but got %s.';
+
+            // Show full error message if the thrown exception is an instance of Error.
+            // See https://github.com/phpspec/phpspec/issues/908
+            if (class_exists('\Error') && $exceptionThrown instanceof \Error) {
```

Comment:
``class_exists``is useless. ``instanceof`` deals fine with non-existent classes (it will obviously return ``false`` in such case)